x/gov/keeper/vote.go

```diff
@@ -68,11 +68,23 @@ func (keeper Keeper) GetVote(ctx sdk.Context, proposalID uint64, voterAddr sdk.A
 	}
 
 	keeper.cdc.MustUnmarshal(bz, &vote)
+
+	// Graceful fallback of deprecated `Option` field, in case there's only 1
+	// VoteOption.
+	if len(vote.Options) == 1 && vote.Options[0].Weight.Equal(sdk.MustNewDecFromStr("1.0")) {
+		vote.Option = vote.Options[0].Option //nolint
+	}
```

Comment:
should we remove the entries in Options?
```
vote.Options = nil
```

Reply:
Also, should we do the same transformation (setting `vote.Option` in `GetVotes`?

Reply:
> Also, should we do the same transformation (setting vote.Option in GetVotes?

Yes, fixed.

> should we remove the entries in Options? vote.Options = nil

I'm not sure. We should for sure populate `Vote.Option` to keep backwards-compatibility, but I would also populate `Vote.Options` too, so that clients know what to use instead of the deprecated field.

Reply:
OK, `Vote.Option` is marked as deprecated, so `Vote.Options` should be used in principal.